styles/Footer.module.scss

```diff
@@ -0,0 +1,315 @@
+.container {
+    background-color: var(--color--light-grey);
+    padding: $grid;
+}
+
+.primaryContent {
+    display: flex;
+    flex-direction: column;
+
+    @include media-query(large) {
+        flex-direction: row;
+    }
+}
+
+.ourLocations {
+    padding-top: $grid * 2;
+    @include media-query(large) {
+        width: 50%;
+        padding: $grid * 2 $grid * 1.5 0px $grid;
+        position: relative;
+    }
+
+    @include media-query(menu-break) {
+        padding: $grid * 2 $grid * 1.5 0px 0px;
+        &::before {
+            content: '';
+            position: absolute;
+            top: 18px;
+            left: -108px;
+            width: 2px;
+            height: 262px;
+            background-color: var(--color--dark-indigo);
+            transform: rotate(30deg);
+        }
+    }
+
+    &Heading {
+        font-weight: $weight--extrabold;
+        @include font-size(ml);
+        margin-top: 0;
+        margin-bottom: $grid;
+    }
+
+    &Location {
+        display: flex;
+        flex-direction: row;
+    }
+
+    &Icon {
+        display: block;
+        width: 32px;
+        margin-right: 5px;
+    }
+
+    &Information {
+        max-width: 90%;
+        // Set to 90% so as to be responsive without the address cropping the SVG icon
```

Comment:
**Nickpick:** It would be nice to avoid this, this element shouldn't clip over the icon regardless, I wonder if using viewBox and svgcrop as mentioned above could help prevent needing this?

Reply:
This is an odd one. I've set a min width on the icon container instead, and now it works without cropping - apparently width wasn't the right css rule here? min width instead of width is new to me...

Reply:
Svgcrop didn't fix it, this was a width and flexbox layout issue

Reply:
I see, yeah it sounds like min-width would do the trick